zlib-ng.h.in

```diff
@@ -29,6 +29,9 @@
   (zlib format), rfc1951 (deflate format) and rfc1952 (gzip format).
 */
 
+/* Almost all content is uninteresting for the VS resource compiler. */
```

Comment:
Perhaps it would read better if it said something along the lines of, "Only expose version information for the VS resource compiler."